px-dropdown-content.html

```diff
@@ -618,10 +621,12 @@
      * the label displayed inside the dropdown.
      */
     _handleDeselection: function(evt) {
-      if(this.multi && this.$.selector.selectedItems.length === 1 && !this.hideSelected) {
-        this.displayValueSelected = this.$.selector.selectedItems[0].innerText.trim();
+      if(this.multi && this.$.selector.selectedItems.length > 0 && this.$.selector.selectedItems.length <= this.displayValueCount && !this.hideSelected) {
```

Comment:
does px have a line length standard? this line & line 590 could possibly be broken up into something shorter.